.github/workflows/checks.yaml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  checks:
+    name: ${{ matrix.cfg.check }}
+    strategy:
+      matrix:
+        cfg:
+          - {check: rake commits, os: ubuntu-18.04, ruby: '2.5'}
+
+    runs-on: ${{ matrix.cfg.os }}
+    steps:
+      - name: Checkout current PR
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Sorry I think I lead you astray with my earlier comment. Fetch depth 0 means [fetch all history for all tags and branches](https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches)! I think we can drop it entirely.

Reply:
Oooh okay got it! I'll remove those now then